src/drivers/px4fmu/fmu.cpp

```diff
@@ -328,10 +332,11 @@ PX4FMU::init()
 		return ret;
 
 	/* try to claim the generic PWM output device node as well - it's OK if we fail at this */
-	_class_instance = register_class_devname(PWM_OUTPUT_BASE_DEVICE_PATH);
+	ret = register_driver(PWM_OUTPUT0_DEVICE_PATH, &fops, 0666, (void *)this);
```

Comment:
this would have registered /dev/pwm_output0 already, and would overflow to /dev/pwm_ouput1 is IO is started. So why hardcode output0 here?